lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -65,22 +65,26 @@ public class BuildUserIdentityHandler
     private final ClientOAuthSessionDetailsService clientOAuthSessionDetailsService;
     private final CiMitService ciMitService;
     private final CiMitUtilityService ciMitUtilityService;
+    private final VerifiableCredentialService verifiableCredentialService;
 
+    //    @SuppressWarnings()
```

Comment:
This shouldn't be in a comment, and it should reference a specific rule (not just suppress everything)

e.g. `@SuppressWarnings("java:S107") // Methods should not have too many parameters`

Reply:
Ah yep I didn't know what number to suppress by though - so was waiting to get another sonarcloud report

Reply:
You can still see the reports if you go to sonar cloud: https://sonarcloud.io/project/pull_requests_list?id=ipv-core-back

(but also, don't delete the sonar comments..!)